### ISSUE TITLE
Avoid accidentally convering #1234 in JIRA to issues in this repo

### ISSUE THREAD

**Issue by retronym** [adriaanm/bbj#7]
Again, this might be a manual post-processing step.

Comments like:

```
SI-9620.md:(PS: #4949 on github is ready to be merged)
```

Should not link to `scala/bug/issue/4949`, but instead point to `scala/scala/pull/4949`.

From the pre-translation:

```
SI-10009.md:this is a regression due to #5141. Before, derived trees for valdefs would be rewritten to have their modifiers reflect that the underlying field is private[this]. After the fields rewrite, this is only done for the modifiers tracked by the tree's symbol. When you call untypecheck, the tree's symbol is dropped and thus it forgets that it's supposed to be private[this].

SI-1048.md:cause unnecessary crashes.  Closes #1048, review by odersky.
SI-1245.md:#4437 seems to be the main focus of activity and handles the whole variety of the problems with asInstanceOf, so I guess this could be closed as a duplicate of SI-4437 before it is forgotten completely?
SI-1318.md:(extempore in [r25151|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25151]) Another one brought to compilation by r25149.  Test case closes #1318, no review.
SI-1336.md:@Jason, since #1893 (the revert) was not accepted, is this still an issue?
SI-1352.md:          // private conflicts with private escape checking - see ticket #3174 for an example.
SI-1431.md:stable via singleton bounds.  Closes #1431, review by odersky.
SI-1432.md:(extempore in [r25279|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25279]) Test case closes #1432, no review.
SI-1439.md:Closes #1439, review by moors.
SI-1448.md:[PR #3294|https://github.com/scala/scala/pull/3294] does this, nothing blows up, one additional {{instanceof}} is required. No feedback from Martin yet.
SI-1528.md:(odersky in [r25127|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25127]) Fixes Zendesk ticket #1528. Review by prokopec.
SI-1697.md:NOTE: the test is WRONG, but it has to be due to #1697/#2337 -- once those are fixed, this one should generate the expected output
SI-1806.md:of #1806, no review.
SI-2089.md:custom hacked compiler. Let's say this closes #2089. Review by odersky.
SI-2094.md:Closes #2094, #2322, #4501. Also, some test cases in neg have been
SI-2094.md:promoted into working programs: #2179, #3774. (They're not in neg for
SI-2094.md:In some cases, such as the original reported ticket in #3528, this
SI-2171.md:(extempore in [r25156|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25156]) Test cases close #2782, #2171, no review.
SI-2179.md:Closes #2094, #2322, #4501. Also, some test cases in neg have been
SI-2179.md:promoted into working programs: #2179, #3774. (They're not in neg for
SI-2179.md:In some cases, such as the original reported ticket in #3528, this
SI-2281.md:(extempore in [r25154|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25154]) Another one I missed which was fixed by r25149.  Closes #2281, no review.
SI-2308.md:Always inflicting arguments upon it was the cause of #2308.
SI-2308.md:Closes #2308, review by moors.
SI-2337.md:NOTE: the test is WRONG, but it has to be due to #1697/#2337 -- once those are fixed, this one should generate the expected output
SI-2593.md:#2540 was closed but this doesn't seem to be addressed; there's still no 'withFilter' on Iterator.
SI-2782.md:(extempore in [r25156|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25156]) Test cases close #2782, #2171, no review.
SI-3048.md:Closes #4110, #3048.  I already ran this by moors, so review by odersky.
SI-3121.md:#3122 is a duplicate of SI-3119
SI-3122.md:#3121 is closely related
SI-3240.md:code involves a self-normalizing type alias.  Closes #3240, review by moors.
SI-3343.md:I guess the bot only recognizes #3343 and not SI-3343.  Fixed in r25110.
SI-3371.md:(extempore in [r25280|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25280]) Test case closes #3371, no review.
SI-3501.md:(rompf in [r25544|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25544]) applying patch provided by Topher, fixes #3501. no review.
SI-3528.md:Closes #2094, #2322, #4501. Also, some test cases in neg have been
SI-3528.md:promoted into working programs: #2179, #3774. (They're not in neg for
SI-3528.md:In some cases, such as the original reported ticket in #3528, this
SI-3708.md:(imaier in [r25495|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25495]) Pulled up the SuperMixin that calls closeOperations for frames into window and let dialogs mix it in as well. Closes #3708.
SI-3774.md:Closes #2094, #2322, #4501. Also, some test cases in neg have been
SI-3774.md:promoted into working programs: #2179, #3774. (They're not in neg for
SI-3774.md:In some cases, such as the original reported ticket in #3528, this
SI-3806.md:#3795, SI-3735
SI-3853.md:(imaier in [r25494|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25494]) Added GraphicsConfiguration to window, frame, dialog constructors. Closes #3853.
SI-3929.md:(imaier in [r25496|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25496]) Allow null components in split pane. Closes #3929.
SI-3960.md:to avoid existential mismatches.  Closes #3960, review by moors.
SI-4110.md:Closes #4110, #3048.  I already ran this by moors, so review by odersky.
SI-4176.md:References #4176, leaving open pending resolution of eta expansion.
SI-4237.md:(extempore in [r25048|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25048]) A getter with type params is still a getter.  Closes #4237.
SI-4262.md:(imaier in [r25493|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25493]) Fixes typo in getter methods. Closes #4262.
SI-4326.md:(prokopec in [r25262|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25262]) Fixes #4326.
SI-4351.md:(prokopec in [r25722|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25722]) Fixes #4351.
SI-4391.md:don't befuddle anyone.  Closes #4391, no review.
SI-4398.md:(prokopec in [r25263|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25263]) Fixes #4398.
SI-4417.md:(prokopec in [r25696|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25696]) Fixes #4417.
SI-4421.md:(pedrofurla in [r25042|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25042]) Adds support for @note in ScalaDoc. Related to #4421. Contributed by Simon Ochsenreither. no review.
SI-4421.md:(kzys in [r25241|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25241]) Adds support for @throws in ScalaDoc. Related to #4421. Contributed by Simon Ochsenreither. Review by dubochet.
SI-4421.md:(kzys in [r25242|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25242]) Add a test for #4421 and clean up a little.
SI-4451.md:(kzys in [r25528|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25528]) Shows modifiers before kinds. Fixes #4451. Contributed by Shows modifiers before kinds. Fixes #4451. Review by heathermiller.
SI-4469.md:debugging code instead. Closes #4469, no review.
SI-4490.md:(kzys in [r25255|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25255]) Fixes some remaining formatting issues. Related to #4490. Contributed by Simon Ochsenreither. Review by ureche.
SI-4501.md:Closes #2094, #2322, #4501. Also, some test cases in neg have been
SI-4501.md:promoted into working programs: #2179, #3774. (They're not in neg for
SI-4501.md:In some cases, such as the original reported ticket in #3528, this
SI-4506.md:The wontfix resolution of ticket #4506 indicates that "def foo" and "def
SI-4512.md:test in r25038, so closes #4512, no review.
SI-4541.md:(prokopec in [r25988|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25988]) Work in progress on #4541.
SI-4547.md:(moors in [r25087|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25087]) closes #4547. parts now also looks in HasMethodMatching-style refinedtype's, so that the argument types of the method we're looking for contribute to the implicit scope.
SI-4553.md:the type before throwing the exception. Closes #4553. I wrote this patch
SI-4556.md:(extempore in [r25040|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25040]) Give some feedback on invalid :type expressions.  Closes #4556, no review.
SI-4561.md:-- Instantaneous repl startup.  Closes #4561.
SI-4578.md:enclosed patch) but we won't let that stop us anymore.  Closes #4578.
SI-4582.md:(extempore in [r25315|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25315]) Test case for ticket #4582.
SI-4584.md:crashers in scalac. Treat it like a hostile witness! Closes #4584, no review.
SI-4595.md:question is in a package.  Closes #4595, no review.
SI-4599.md:(extempore in [r24998|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=24998]) Fix #4599: XMLEventReader issue with input stream.
SI-4601.md:(extempore in [r25120|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25120]) Test case closes #4601, no review.
SI-4603.md:(odersky in [r25394|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25394]) Closes #4603. Review by extempore.
SI-4614.md:(extempore in [r25119|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25119]) Fix broken tool-windows.tmpl.  Closes #4614, no review.
SI-4616.md:This was done at r23666, to fix #4001. Here's the log:
SI-4616.md:    immutable.Map keySet returns immutable.Set.  Closes #4001, no review.
SI-4636.md:chance of working.  Closes #4636, no review.
SI-4641.md:(kzys in [r25077|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25077]) The '#' sign may easily to lead to problems, So we use '_' instead of '#'. Closes #4641.
SI-4642.md:Closes #4642, no review.
SI-4656.md:Closes #4656, no review.
SI-4658.md://#4658
SI-4659.md:(extempore in [r25122|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25122]) Deprecated BigInt#~ in favor of unary_~.  Closes #4659, no review.
SI-4660.md:(extempore in [r25074|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25074]) Fix for wrong results from groupBy on views.  Closes #4660, no review.
SI-4661.md:promote it from page 14 to page 11.  Closes #4661, no review.
SI-4662.md:is created via getLines.  Closes #4671, #4662, no review.
SI-4671.md:is created via getLines.  Closes #4671, #4662, no review.
SI-4678.md:(prokopec in [r25112|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25112]) Fixed an overflow which occurs in hashtable size computations. Fixes #4678.
SI-4680.md:References #4680.  Review by odersky.
SI-4692.md:(moors in [r25092|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25092]) closes #4692: unification in type constructor inference now widens *and* dealiases when necessary
SI-4696.md:Closes #490, #896, #4696.  Review by moors.  (Note: I think this is
SI-4697.md:performance.  Closes #4697, no review.
SI-4708.md:(extempore in [r25123|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25123]) Removed an unused method.  Closes #4708, no review.
SI-4709.md:(prokopec in [r25261|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25261]) Fixes #4709.
SI-4710.md:Closes #4710, no review.
SI-4712.md:(odersky in [r25126|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25126]) Fixes #4712. Review by moors.
SI-4713.md:(odersky in [r25128|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25128]) Putting in a fix for #4713. Not sure this goes deep enough, though. Review and maybe deeper changes by plocinic.
SI-4716.md:(prokopec in [r25742|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25742]) Fixes #4716.
SI-4721.md:From the comments of #2308.
SI-4723.md:(prokopec in [r25260|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25260]) Fixes #4723.
SI-4727.md:(extempore in [r25136|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25136]) Test case closes #4727, no review.
SI-4727.md:(extempore in [r25139|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25139]) Moved test for #4727 to where it was supposed to go, no review.
SI-4731.md:caught, but for now we can have a sanity check.  Closes #4731,
SI-4737.md:(extempore in [r25244|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25244]) Test case for #4737, no review.
SI-4748.md:(extempore in [r25209|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25209]) Fix for unfortunate bug in Range.init.  Closes #4748, no review.
SI-4752.md:Closes #4752, no review.
SI-4753.md:classOf, so I knew what was up. Closes #4753, review by odersky.
SI-4757.md:seeks improvement.  Closes #4757, review by odersky.
SI-4757.md:(extempore in [r25220|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25220]) Test case for #4757, no review.
SI-4761.md:(prokopec in [r25258|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25258]) Fixes #4761.
SI-4764.md:Closes #4764, review by rompf.
SI-4784.md:I checked again with the prebuilt nightly and indeed, no problem there. It turns out that in the build I used previously, I had disabled the check that prevents loading {{XX$class.class}} files (see #4767). Consequently, using the nightly, this one fails for {{scalac -Yinline}} (curiously enough not for {{-optimize}} ??).
SI-4791.md:Fixes #4791. No review.
SI-4792.md:Fixes #4792. No review.
SI-4799.md:(imaier in [r25497|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25497]) Added some missing methods to ListView. Closes #4799.
SI-4811.md:(imaier in [r25492|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25492]) Fixes typo in setter method. Closes #4811.
SI-4823.md:(imaier in [r25491|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25491]) Applied patch by Gordon Tyler. Closes #4823.
SI-4894.md:(prokopec in [r25748|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25748]) Fixes #4894.
SI-4895.md:(prokopec in [r25752|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25752]) Fixes #4895.
SI-490.md:Closes #490, #896, #4696.  Review by moors.  (Note: I think this is
SI-4902.md:(imaier in [r25490|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25490]) Install mouse moves listeners lazily. Closes #4902.
SI-4925.md:        definitions.getClass(name) // see tickets #2464, #3756
SI-5127.md:(odersky in [r25913|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=25913]) Closes #5127. Review by extempore.
SI-5230.md:(odersky in [r26085|https://codereview.scala-lang.org/fisheye/changelog/scala-svn?cs=26085]) Attempt to fix #5230. Review by burmako.
SI-5330.md:{(f#7469: A#7103 => Any#3533)Unit#1690}.asSeenFrom(xs#8667.type, trait M#9)
SI-5330.md:tp1 = (f#8670: _$1#8669 => Any#3533)Unit#1690
SI-5330.md:result = (f#8673: _$1#8671 => Any#3533)Unit#1690
SI-5335.md:}{<null>}.x#2776{<null>}
SI-5335.md:See this x#2776? It should've been erased, but it wasn't. The problem is that the definition of x is in a block that we don't recur into. I believe, we need a separate run that collects local symbols - that would be more robust. Is the imposed performance cost acceptable?
SI-5399.md:Closed #5400 as a duplicate since it regressed in the same commit.  Here's the source for an additional test case.
SI-5415.md: found   : Test2.CoffeesTable#1915 => Boolean#196
SI-5415.md: required: Test2#1910.CoffeesTable#1915 => Boolean#196
SI-5415.md: found   : Test2#1909#MC.CoffeesTable#1915#TR => Boolean#196#CLS
SI-5415.md: required: Test2#1910#FV.CoffeesTable#1915#TR => Boolean#196#CLS
SI-5432.md:Would one of you terribly mind explaining to me in a bit more detail what the concern is around this ad-hoc solution? Not challenging the decision—I'm quite happy to acknowledge that your instincts are much more likely to be right than mine on this—but I'd like to further develop my own sense of good-Scala/bad-Scala, and I still don't see the relevant difference between {{orderingToPartialOrdering}} and {{Seq.apply}} producing {{List}}&#8203;s. Is there some backwards compatibility issue that I'm missing? Or is it just a general principle of not wanting to invest any sort of effort in addressing a part of the problem if you're currently investing in solving the whole problem?
SI-5489.md:val tp = refinedType(info.parents, owner) // Object#1815
SI-5489.md:val thistp = tp.typeSymbol.thisType       // Object#1815
SI-5489.md:           //  def <init>#10195(arg$outer#16869: repro.CausesProblems#7101): $anon#10194;
SI-5489.md:           //  override def blerg#10196(): String#1899;
SI-5489.md:           //  val repro$CausesProblems$$anon$$$outer#11591(): repro.CausesProblems#7101;
SI-5489.md:           //  private[this] val $outer#11592: repro.CausesProblems#7101
SI-5489.md:addMember(Object#1815, Object#1815, blerg#10196)
SI-5489.md:addMember(Object#1815, Object#1815, repro$CausesProblems$$anon$$$outer#11591)
SI-5496.md:macroExpand: Macros#12.Shmacros#9529.foo#9531(2)
SI-5496.md:looking for macro implementation: Macros#13.Shmacros#9528.foo#9531
SI-5496.md:found implementation at: Macros#13.Shmacros#9528.defmacro$foo#9533
SI-5496.md:macro is not overridden: Macros#12.Shmacros#9529.foo#9531
SI-5545.md:new info:  lang#2520.this.Object#2592
SI-5545.md:        with <empty>#4.this.F#8[T1#7693,R#7694] {
SI-5545.md:  signature after erasure: (<param> <triedcooking> v1#19237: <root>#2#scala#30#Int#453)<root>#2#java#26#lang#2520#Object#2592
SI-5789.md:[log erasure(->posterasure ->posterasure)] Creating implClass for trait Function2#1114
SI-5789.md:[log erasure(->posterasure ->posterasure)] trait Function2#1114.implClass already exists: class Function2$class#787
SI-5789.md:mixinImplClassMembers(mixinClass = class Function2$class#787, mixinInterface = trait Function2#1114)
SI-5789.md:-          // the IMPLCLASS flag is unset, I believe.) See ticket #4285.
SI-5900.md:*   acd7780 Merge pull request #3014 from ceedubs/pr/implicitNotFound-scaladoc
SI-5900.md:* |   90a3126 Merge pull request #3005 from paulp/pr/7886
SI-5903.md:Block[1](List(ValDef[2](Modifiers(), newTermName("scrut"), TypeTree[3](), Ident[3](newTermName("<unapply-selector>")#20755)), ValDef[2](Modifiers(), newTermName("part1"), TypeTree[3](), Apply[3](Select[4](Ident[5](newTermName("scrut")#20758), newTermName("$div")#17275), List(Literal[6](Constant(2))))), ValDef[2](Modifiers(), newTermName("part2"), TypeTree[3](), Apply[3](Select[4](Ident[5](newTermName("scrut")#20758), newTermName("$div")#17275), List(Literal[6](Constant(2)))))), Apply[1](TypeApply[7](Select[8](Select[9](Ident[10](scala#20), scala.Some#1309), newTermName("apply")#20794), List(TypeTree[11]())), List(Apply[11](TypeApply[12](Select[13](Select[14](Ident[10](scala#20), scala.Tuple2#1513), newTermName("apply")#19908), List(TypeTree[3](), TypeTree[3]())), List(Ident[3](newTermName("part1")#20759), Ident[3](newTermName("part2")#20760))))))
SI-5903.md:[1] TypeRef(ThisType(scala#21), scala.Some#1308, List(TypeRef(ThisType(scala#21), scala.Tuple2#1512, List(TypeRef(ThisType(scala#21), scala.Int#849, List()), TypeRef(ThisType(scala#21), scala.Int#849, List())))))
SI-5903.md:[7] MethodType(List(newTermName("x")#20886), TypeRef(ThisType(scala#21), scala.Some#1308, List(TypeRef(ThisType(scala#21), scala.Tuple2#1512, List(TypeRef(ThisType(scala#21), scala.Int#849, List()), TypeRef(ThisType(scala#21), scala.Int#849, List()))))))
SI-5903.md:[8] PolyType(List(newTypeName("A")#20795), MethodType(List(newTermName("x")#20796), TypeRef(ThisType(scala#21), scala.Some#1308, List(TypeRef(NoPrefix, newTypeName("A")#20795, List())))))
SI-5903.md:[9] SingleType(SingleType(ThisType(<root>#2), scala#20), scala.Some#1309)
SI-5903.md:[11] TypeRef(ThisType(scala#21), scala.Tuple2#1512, List(TypeRef(ThisType(scala#21), scala.Int#849, List()), TypeRef(ThisType(scala#21), scala.Int#849, List())))
SI-5903.md:[12] MethodType(List(newTermName("_1")#20884, newTermName("_2")#20885), TypeRef(ThisType(scala#21), scala.Tuple2#1512, List(TypeRef(ThisType(scala#21), scala.Int#849, List()), TypeRef(ThisType(scala#21), scala.Int#849, List()))))
SI-5903.md:[13] PolyType(List(newTypeName("T1")#19909, newTypeName("T2")#19910), MethodType(List(newTermName("_1")#19911, newTermName("_2")#19912), TypeRef(ThisType(scala#21), scala.Tuple2#1512, List(TypeRef(NoPrefix, newTypeName("T1")#19909, List()), TypeRef(NoPrefix, newTypeName("T2")#19910, List())))))
SI-5903.md:[14] SingleType(SingleType(ThisType(<root>#2), scala#20), scala.Tuple2#1513)
SI-5932.md:args = List(Test#20.this.x2#9255)
SI-5932.md:underlyingClass(args.head.tpe) = Test#9258 // RefinementClassSymbol
SI-5954.md:*** package$buhB#6946
SI-5954.md:*** buhB#7260
SI-5954.md:assertion failed: Symbol is unexpectedly overloaded with alternatives 'object package$buhB#6946#MOD (object package$buhB#6946#MOD->package buhbase#29#PK->package <root>#2#PK)'and'object package$buhB#7260#MOD (object package$buhB#7260#MOD->package buhbase#29#PK->package <root>#2#PK)'.
SI-5954.md:So, Namer tried to look up buhB, didn't find it, created #7260. Later #7260 was renamed to package$buhB and by then the damage was done. I don't yet know where the name gets changed.
SI-6035.md:          // that it isn't right here. See #4757 for the immediate
SI-6187.md:  object Test#6954 extends Object#1728 with App#35 {
SI-6187.md:    def <init>#7036(): Test#6955.type = {
SI-6187.md:      Test#6955.super.<init>#4684();
SI-6187.md:    def m#7037[T#7038 >: Nothing#3312 <: Any#3311](t#11882: T#7039): List#2441[List#2441[T#7038]] = immutable#4655.this.List#10904.apply#11945[List#2441[T#7039]](immutable#4655.this.List#10904.apply#11945[(T#7039, Boolean#137)](scala#20.Tuple2#1515.apply#12675[T#7039, Boolean#137](t#11882, true)).collect#12791[T#7039, List#2441[T#7039]](({
SI-6187.md:      final <synthetic> class $anonfun#22112 extends scala#21.runtime#1626.AbstractPartialFunction#2953[(T#7039, Boolean#137),T#7039] with Serializable#1295 {
SI-6187.md:        final override def applyOrElse#22114[A1#22115 >: Nothing#3312 <: (T#7039, Boolean#137), B1#22116 >: T#7039 <: Any#3311](x$1#22124: A1#22115, default#22125: A1#22115 => B1#22116): B1#22116 = (x$1#22124: Any#3311) match {
SI-6187.md:          case (_1#15877: T#7039, _2#15878: Boolean#137)(T#7039, Boolean#137)((x#22136 @ _), true) => x#22136
SI-6187.md:        final def isDefinedAt#22119(x$1#22139: (T#7039, Boolean#137)): Boolean#137 = (x$1#22139: (T#7039, Boolean#137) @unchecked#1577) match {
SI-6187.md:          case (_1#15884: T#7039, _2#15885: Boolean#137)(T#7039, Boolean#137)((x#22148 @ _), true) => true
SI-6187.md:    }: PartialFunction#1007[(T#7039, Boolean#137),T#7039]))(immutable#4655.this.List#10904.canBuildFrom#11939[T#7039]))
SI-6187.md:   symbol definition: final class Array#131[T#11679 >: ? <: ?] extends Object#1728
SI-6187.md:                 tpe: Array#131[Object#1728]
SI-6187.md:      context owners: value x4#22481 -> method applyOrElse#22114 -> anonymous class $anonfun$m$1#22112 -> method m#7037 -> object Test#6955 -> package <empty>#3
SI-6187.md:DefDef( // final override def applyOrElse#22114(x$1#32175: Tuple2#1514,default#32176: Function1#416): Object#1728
SI-6187.md:  ValDef( // x$1#22124: Tuple2#1514
SI-6187.md:    <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:  <tpt> // tree.tpe=Object#1728
SI-6187.md:  Block( // tree.tpe=Object#1728
SI-6187.md:    ValDef( // case val x1#22476: Object#1728
SI-6187.md:      <tpt> // tree.tpe=Object#1728
SI-6187.md:      Typed( // tree.tpe=Object#1728
SI-6187.md:        "x$1" // x$1#22124: Tuple2#1514, tree.tpe=Tuple2#1514
SI-6187.md:        <tpt> // tree.tpe=Object#1728
SI-6187.md:    LabelDef( // case def case7#22485(): Object#1728, tree.tpe=Object#1728
SI-6187.md:      If( // tree.tpe=Object#1728
SI-6187.md:        Apply( // final def $isInstanceOf#4720[T0#4721 >: ? <: ?](): Boolean#137 in class Object#1728, tree.tpe=Boolean#137
SI-6187.md:          TypeApply( // final def $isInstanceOf#4720[T0#4721 >: ? <: ?](): Boolean#137 in class Object#1728, tree.tpe=()Boolean#137
SI-6187.md:            "x1"."$isInstanceOf" // final def $isInstanceOf#4720[T0#4721 >: ? <: ?](): Boolean#137 in class Object#1728, tree.tpe=[T0#4721 >: ? <: ?]()Boolean#137
SI-6187.md:            <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:        Block( // tree.tpe=Object#1728
SI-6187.md:          ValDef( // val x3#22478: Tuple2#1514
SI-6187.md:            <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:            Typed( // tree.tpe=Tuple2#1514
SI-6187.md:              Apply( // final def $asInstanceOf#4722[T0#4723 >: ? <: ?](): T0#4723 in class Object#1728, tree.tpe=Tuple2#1514
SI-6187.md:                TypeApply( // final def $asInstanceOf#4722[T0#4723 >: ? <: ?](): T0#4723 in class Object#1728, tree.tpe=()Tuple2#1514
SI-6187.md:                  "x1"."$asInstanceOf" // final def $asInstanceOf#4722[T0#4723 >: ? <: ?](): T0#4723 in class Object#1728, tree.tpe=[T0#4723 >: ? <: ?]()T0#4723
SI-6187.md:                  <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:              <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:          If( // tree.tpe=Object#1728
SI-6187.md:            Apply( // final def ne#4710(x$1#34965: Object#1728): Boolean#137 in class Object#1728, tree.tpe=Boolean#137
SI-6187.md:              "x3"."ne" // final def ne#4710(x$1#34965: Object#1728): Boolean#137 in class Object#1728, tree.tpe=(x$1#34965: Object#1728)Boolean#137
SI-6187.md:            Block( // tree.tpe=Object#1728
SI-6187.md:              ValDef( // val x#22136: Object#1728
SI-6187.md:                <tpt> // tree.tpe=Object#1728
SI-6187.md:                Apply( // val _1#12646(): Object#1728 in class Tuple2#1514, tree.tpe=Object#1728
SI-6187.md:                  "x3"."_1" // val _1#12646(): Object#1728 in class Tuple2#1514, tree.tpe=()Object#1728
SI-6187.md:                Apply( // val _2$mcZ$sp#26445(): Boolean#137 in class Tuple2#1514, tree.tpe=Boolean#137
SI-6187.md:                  "x3"."_2$mcZ$sp" // val _2$mcZ$sp#26445(): Boolean#137 in class Tuple2#1514, tree.tpe=()Boolean#137
SI-6187.md:              If( // tree.tpe=Object#1728
SI-6187.md:                Block( // tree.tpe=Object#1728
SI-6187.md:                  Apply( // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=Object#1728
SI-6187.md:                    "matchEnd6" // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=(x#34967: Object#1728)Object#1728
SI-6187.md:                    "x" // val x#22136: Object#1728, tree.tpe=Object#1728
SI-6187.md:                Apply( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:                  "case8" // case def case8#22486(): Object#1728, tree.tpe=()Object#1728
SI-6187.md:            Apply( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:              "case8" // case def case8#22486(): Object#1728, tree.tpe=()Object#1728
SI-6187.md:        Apply( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:          "case8" // case def case8#22486(): Object#1728, tree.tpe=()Object#1728
SI-6187.md:    LabelDef( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:      Apply( // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=Object#1728
SI-6187.md:        "matchEnd6" // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=(x#34967: Object#1728)Object#1728
SI-6187.md:        Apply( // def apply#14937(v1#32182: Object#1728): Object#1728 in trait Function1#416, tree.tpe=Object#1728
SI-6187.md:          "default"."apply" // def apply#14937(v1#32182: Object#1728): Object#1728 in trait Function1#416, tree.tpe=(v1#32182: Object#1728)Object#1728
SI-6187.md:          "x$1" // x$1#15870: Tuple2#1514, tree.tpe=Tuple2#1514
SI-6187.md:    LabelDef( // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=Object#1728
SI-6187.md:      "x" // x#22484: Object#1728, tree.tpe=Object#1728
SI-6187.md:      "x" // x#22484: Object#1728, tree.tpe=Object#1728
SI-6187.md:    final class Array#131[T#11679 >: ? <: ?] extends Object#1728
SI-6187.md:  args = List(TypeRef(TypeSymbol(class Object#1728 extends )))
SI-6187.md:   symbol definition: final class Array#131[T#11679 >: ? <: ?] extends Object#1728
SI-6187.md:                 tpe: Array#131[Object#1728]
SI-6187.md:      context owners: value x4#22481 -> method applyOrElse#22114 -> anonymous class $anonfun$m$1#22112 -> method m#7037 -> object Test#6955 -> package <empty>#3
SI-6187.md:DefDef( // final override def applyOrElse#22114(x$1#32175: Tuple2#1514,default#32176: Function1#416): Object#1728
SI-6187.md:  ValDef( // x$1#22124: Tuple2#1514
SI-6187.md:    <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:  <tpt> // tree.tpe=Object#1728
SI-6187.md:  Block( // tree.tpe=Object#1728
SI-6187.md:    ValDef( // case val x1#22476: Object#1728
SI-6187.md:      <tpt> // tree.tpe=Object#1728
SI-6187.md:      Typed( // tree.tpe=Object#1728
SI-6187.md:        "x$1" // x$1#22124: Tuple2#1514, tree.tpe=Tuple2#1514
SI-6187.md:        <tpt> // tree.tpe=Object#1728
SI-6187.md:    LabelDef( // case def case7#22485(): Object#1728, tree.tpe=Object#1728
SI-6187.md:      If( // tree.tpe=Object#1728
SI-6187.md:        Apply( // final def $isInstanceOf#4720[T0#4721 >: ? <: ?](): Boolean#137 in class Object#1728, tree.tpe=Boolean#137
SI-6187.md:          TypeApply( // final def $isInstanceOf#4720[T0#4721 >: ? <: ?](): Boolean#137 in class Object#1728, tree.tpe=()Boolean#137
SI-6187.md:            "x1"."$isInstanceOf" // final def $isInstanceOf#4720[T0#4721 >: ? <: ?](): Boolean#137 in class Object#1728, tree.tpe=[T0#4721 >: ? <: ?]()Boolean#137
SI-6187.md:            <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:        Block( // tree.tpe=Object#1728
SI-6187.md:          ValDef( // val x3#22478: Tuple2#1514
SI-6187.md:            <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:            Typed( // tree.tpe=Tuple2#1514
SI-6187.md:              Apply( // final def $asInstanceOf#4722[T0#4723 >: ? <: ?](): T0#4723 in class Object#1728, tree.tpe=Tuple2#1514
SI-6187.md:                TypeApply( // final def $asInstanceOf#4722[T0#4723 >: ? <: ?](): T0#4723 in class Object#1728, tree.tpe=()Tuple2#1514
SI-6187.md:                  "x1"."$asInstanceOf" // final def $asInstanceOf#4722[T0#4723 >: ? <: ?](): T0#4723 in class Object#1728, tree.tpe=[T0#4723 >: ? <: ?]()T0#4723
SI-6187.md:                  <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:              <tpt> // tree.tpe=Tuple2#1514
SI-6187.md:          If( // tree.tpe=Object#1728
SI-6187.md:            Apply( // final def ne#4710(x$1#34965: Object#1728): Boolean#137 in class Object#1728, tree.tpe=Boolean#137
SI-6187.md:              "x3"."ne" // final def ne#4710(x$1#34965: Object#1728): Boolean#137 in class Object#1728, tree.tpe=(x$1#34965: Object#1728)Boolean#137
SI-6187.md:            Block( // tree.tpe=Object#1728
SI-6187.md:              ValDef( // val x#22136: Object#1728
SI-6187.md:                <tpt> // tree.tpe=Object#1728
SI-6187.md:                Apply( // val _1#12646(): Object#1728 in class Tuple2#1514, tree.tpe=Object#1728
SI-6187.md:                  "x3"."_1" // val _1#12646(): Object#1728 in class Tuple2#1514, tree.tpe=()Object#1728
SI-6187.md:                Apply( // val _2$mcZ$sp#26445(): Boolean#137 in class Tuple2#1514, tree.tpe=Boolean#137
SI-6187.md:                  "x3"."_2$mcZ$sp" // val _2$mcZ$sp#26445(): Boolean#137 in class Tuple2#1514, tree.tpe=()Boolean#137
SI-6187.md:              If( // tree.tpe=Object#1728
SI-6187.md:                Block( // tree.tpe=Object#1728
SI-6187.md:                  Apply( // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=Object#1728
SI-6187.md:                    "matchEnd6" // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=(x#34967: Object#1728)Object#1728
SI-6187.md:                    "x" // val x#22136: Object#1728, tree.tpe=Object#1728
SI-6187.md:                Apply( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:                  "case8" // case def case8#22486(): Object#1728, tree.tpe=()Object#1728
SI-6187.md:            Apply( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:              "case8" // case def case8#22486(): Object#1728, tree.tpe=()Object#1728
SI-6187.md:        Apply( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:          "case8" // case def case8#22486(): Object#1728, tree.tpe=()Object#1728
SI-6187.md:    LabelDef( // case def case8#22486(): Object#1728, tree.tpe=Object#1728
SI-6187.md:      Apply( // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=Object#1728
SI-6187.md:        "matchEnd6" // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=(x#34967: Object#1728)Object#1728
SI-6187.md:        Apply( // def apply#14937(v1#32182: Object#1728): Object#1728 in trait Function1#416, tree.tpe=Object#1728
SI-6187.md:          "default"."apply" // def apply#14937(v1#32182: Object#1728): Object#1728 in trait Function1#416, tree.tpe=(v1#32182: Object#1728)Object#1728
SI-6187.md:          "x$1" // x$1#15870: Tuple2#1514, tree.tpe=Tuple2#1514
SI-6187.md:    LabelDef( // case def matchEnd6#22483(x#34967: Object#1728): Object#1728, tree.tpe=Object#1728
SI-6187.md:      "x" // x#22484: Object#1728, tree.tpe=Object#1728
SI-6187.md:      "x" // x#22484: Object#1728, tree.tpe=Object#1728
SI-6187.md:    final class Array#131[T#11679 >: ? <: ?] extends Object#1728
SI-6187.md:  args = List(TypeRef(TypeSymbol(class Object#1728 extends )))
SI-6187.md:error: java.lang.IllegalArgumentException: Could not find proxy for default#15872: Function1#416 in List(value default#15872, method applyOrElse#15868, anonymous class $anonfun#15861, method m#7037, object Test#6955, package <empty>#3, package <root>#2) (currentOwner= method applyOrElse#22114 )
SI-6187.md:Exception in thread "main" java.lang.IllegalArgumentException: Could not find proxy for default#15872: Function1#416 in List(value default#15872, method applyOrElse#15868, anonymous class $anonfun#15861, method m#7037, object Test#6955, package <empty>#3, package <root>#2) (currentOwner= method applyOrElse#22114 )
SI-6317.md:error: java.lang.IllegalArgumentException: Could not find proxy for case val x1#23793: Tuple2#1518 in List(value x1#23793, method apply#27718, anonymous class $anonfun$1#27717, value <local ScalaCompilerKiller>#12215, class ScalaCompilerKiller#6974, package <empty>#3, package <root>#2)  (currentOwner= List(value r#16190, value $anonfun#16186, value <local ScalaCompilerKiller>#12215, class ScalaCompilerKiller#6974, package <empty>#3, package <root>#2) )
SI-6359.md:F class M#7016 [+final]
SI-6359.md:*     method lazyString#7392
SI-6359.md:class M#7016 (final)
SI-6359.md:      method lazyString#7392
SI-6359.md:  object M#7393 (<synthetic>)
SI-6359.md:  object M#7394 (<synthetic>)
SI-6359.md:O         class Y#12575 [Owner was method lazyString#7392, now method extension$lazyString#13933]
SI-6359.md:O         object X#12573 [Owner was method lazyString#7392, now method extension$lazyString#13933]
SI-6359.md:O         value $anonfun#12580 [Owner was method lazyString#7392, now method extension$lazyString#13933] (<synthetic>)
SI-6436.md:git: branch = 2.10.x, commit = 1d072fcb41 Merge pull request #1408 from scalamacros/topic/direct-test (Paul Phillips, 26.09.2012 22:54:01 -07:00)
SI-6442.md:Pull request #1425 removes the `MigrationSystem` and adds an `ActorDSL` object which also exists in Akka. The pull request also fixes all the "actmig" test cases to use `ActorDSL`.
SI-6493.md:<method> def foo#7445(): scala#21.this.AnyRef#2222{type Bar#12153 <: scala#21.this.AnyRef#2222}#Bar#12125
SI-6493.md:<method> def foo#7445(): scala#21.this.AnyRef#2222{type Bar#12153 <: scala#21.this.AnyRef#2222}#Bar#12123
SI-6493.md:    // #3731: return sym1 for which holds: pre bound sym.name to sym and
SI-6522.md:where val candidateUpperObject: CanonicalGeneration#9908.this.Upper#44230
SI-6563.md:    Merge pull request #1125 from hubertp/2.10.x-issue/6214
SI-6648.md:  object transformer1#6962
SI-6648.md:* object transformer1#6963
SI-6665.md:[1] TypeRef(ThisType(scala#21), scala.Nothing#3310, List())
SI-6665.md:[2] TypeRef(ThisType(scala#21), scala.Function1#776, List(TypeRef(ThisType(scala#21), scala.Int#317, List()), TypeRef(ThisType(scala#21), scala.Unit#1520, List())))
SI-6665.md:[5] TypeRef(ThisType(scala#21), scala.Unit#1520, List())
SI-6665.md:[6] MethodType(List(newTermName("n")#12571), TypeRef(ThisType(scala#21), scala.Unit#1520, List()))
SI-6665.md:[7] ThisType(Manual#6947)
SI-6692.md:    improvement. Closes #4757, review by odersky.
SI-6990.md:0512 [9] TypeRef(ThisType(scala#19), scala.Any#3301, List())
SI-7120.md:// derived from a real-world Slick app (support #3035)
SI-7261.md:         0: getstatic     #1296               // Field scala/tools/nsc/settings/AbsScalaSettings$.MODULE$:Lscala/tools/nsc/settings/AbsScalaSettings$;
SI-7261.md:         4: invokevirtual #1219               // Method settings:()Lscala/tools/nsc/Settings;
SI-7261.md:         7: invokeinterface #1312,  1         // InterfaceMethod scala/tools/nsc/settings/ScalaSettings.developer:()Lscala/tools/nsc/settings/MutableSettings$BooleanSetting;
SI-7261.md:        12: invokevirtual #1305               // Method scala/tools/nsc/settings/AbsScalaSettings$.valueOfBooleanSetting:(Lscala/tools/nsc/settings/AbsSettings$AbsSetting;)Z
SI-7261.md:        18: getstatic     #1296               // Field scala/tools/nsc/settings/AbsScalaSettings$.MODULE$:Lscala/tools/nsc/settings/AbsScalaSettings$;
SI-7261.md:        22: invokevirtual #1219               // Method settings:()Lscala/tools/nsc/Settings;
SI-7261.md:        25: invokeinterface #1301,  1         // InterfaceMethod scala/tools/nsc/settings/ScalaSettings.debug:()Lscala/tools/nsc/settings/MutableSettings$BooleanSetting;
SI-7261.md:        30: invokevirtual #1305               // Method scala/tools/nsc/settings/AbsScalaSettings$.valueOfBooleanSetting:(Lscala/tools/nsc/settings/AbsSettings$AbsSetting;)Z
SI-7300.md:I think this one can be closed since the tests are merged too (PR [#2355|https://github.com/scala/scala/pull/2355]). 
SI-7304.md:(From typesafe ticket #2046)
SI-7598.md:  class Outer$Inner#8101 extends Object#145 {
SI-7598.md:    <synthetic> <paramaccessor> <artifact> protected val $outer#15239: Outer#7816 = _;
SI-7598.md:    <synthetic> <stable> <artifact> def $outer#15238(): Outer#7816 = Outer$Inner#8101.this.$outer#15239;
SI-7598.md:    def <init>#13449($outer#15240: Outer#7816): Outer$Inner#8101 = {
SI-7598.md:      if ($outer#15240.eq#5861(null))
SI-7598.md:        Outer$Inner#8101.this.$outer#15239 = $outer#15240;
SI-7598.md:      Outer$Inner#8101.super.<init>#3011();
SI-7598.md:      null.eq#5861(Outer#7816.this.X#8099());
SI-7603.md:// need to be lazy, #1782. beforeTyper to allow inferView in annotation args, SI-5892.
SI-7649.md:Unpositioned tree #2392
SI-7736.md:WeakTypeTag[A#7080#TPE]
SI-7736.md:  import scala#20#PK.language#861#MOD.experimental#6997#MOD.macros;
SI-7736.md:  object Test#6978#MOD extends AnyRef#2169#TPE with App#35#TRT {
SI-7736.md:    def <init>#7078#PCTOR(): Test#6979#MODC.type = {
SI-7736.md:      Test#6979#MODC.super.<init>#4691#PCTOR();
SI-7736.md:    def foo#7079#METH[A#7080#TPE >: Nothing#3304#CLS <: Any#3303#CLS]: AnyRef#2169#TPE{def bar#16748#MAC[B#16749#TPE]: Unit#1595#CLS} = {
SI-7736.md:      final class $anon#11933#AC extends scala#20#PK.AnyRef#2169#TPE {
SI-7736.md:          $anon#11933#AC.super.<init>#4691#PCTOR();
SI-7736.md:        @scala#21#PK.reflect#1622#PK.macros#2309#PK.internal#16551#PK.macroImpl#16732#CLS(`macro`("versionFormat" = 1, "className" = "Macros", "methodName" = "impl", "signature" = List#10952#MOD(-1, 0, 1))[A#7081#TPE#SKO, B#11937#TPE#SKO]) <macro> def bar#11935#MAC[B#11936#TPE >: Nothing#3304#CLS <: Any#3303#CLS]: Unit#1595#CLS = Macros#6974#MOD.impl#16439#METH[A#7081#TPE#SKO, B#11937#TPE#SKO]
SI-7818.md:tree.tpe         = Observable1#7037[_$1#12344]
SI-7818.md:<_$1#12344>.info =  <: T#7040
SI-7818.md:pt               = Observable1#7037[T#15644]
SI-7851.md:    Merge pull request #2935 from densh/topic/si-7304-6489-6701
SI-7851.md:   1:	invokevirtual	#1067; //Method scala$tools$nsc$transform$patmat$PatternMatching$OptimizingMatchTranslator$$$outer:()Lscala/tools/nsc/transform/patmat/PatternMatching;
SI-7851.md:   1:	invokevirtual	#1066; //Method scala$tools$nsc$transform$patmat$PatternMatching$OptimizingMatchTranslator$$$outer:()Lscala/tools/nsc/transform/patmat/PatternMatching;
SI-7907.md:    Merge pull request #3030 from xeno-by/topic/fundep-views
SI-8101.md:[info] While validating #2018
SI-8101.md:[info] [L   1        ] #2018   [0:14397]       PackageDef // account
SI-8101.md:[info]   [L  34 P# 2018] #2017   [1135:14397]    ModuleDef  // Info extends Logger {
SI-8114.md:// derived from a real-world Slick app (support #3035)
SI-8128.md:This is support ticket #3045
SI-8166.md:// derived from a real-world Slick app (support #3035)
SI-8254.md:      http://docs.oracle.com/javase/6/docs/platform/serialization/spec/input.html#2971 

SI-8337.md:    Merge pull request #2370 from retronym/ticket/7319-2
SI-8369.md:    Merge pull request #3305 from xeno-by/topic/copy-untyped
SI-8369.md:sandbox/test.scala:3: error: type arguments [S#7739] do not conform to method read$default$1#14733's type parameter bounds [S#14734 <: Bug#7603.Sys#7735[S#7741]]
SI-8394.md:I opened a PR for this: [#4433|https://github.com/scala/scala/pull/4433]
SI-8439.md:      // don't be too zealous with the exceptions, see #2641
SI-8777.md:Note the reference to `Okay#7757.this.TreePrinter#14744` in the last line above.
SI-8899.md:Making {{filterImpl}} non-private in [PR #3949|https://github.com/scala/scala/pull/3949] was indeed not binary compatible. The problem is that making a trait method non-private changes its calling convention. Here's an example:
SI-8999.md:Originated in [scala-js/scala-js#1204|https://github.com/scala-js/scala-js/issues/1204].
SI-9113.md:I have opened [pr #4266|https://github.com/scala/scala/pull/4266], which fixes this issue.
SI-9113.md:The previous PR had some problems with jenkins, so I have closed it and opened [#4267|https://github.com/scala/scala/pull/4267] in its place.
SI-9546.md:If it helps, this regressed in [f5dc96b|https://github.com/retronym/scala/commit/f5dc96bb550a840ea150985125f52e025ac8ac49] ([#4675|https://github.com/scala/scala/pull/4675]).
SI-9602.md:@retronym the #3992 PR seems a lot more ambitious (and correspondingly controversial, side-effect-ful, etc) than what I'm after.   
SI-9620.md:(PS: #4949 on github is ready to be merged)
SI-9620.md:Fixed in 2.12.0-M4 ([#4952|https://github.com/scala/scala/pull/4952]) by [~felixmulder], closing.
SI-9693.md:In accordance with the spec, and under both outer ref check implementations (before and after #4974), this prints:
```
